server/src/test/java/org/elasticsearch/ingest/IngestSourceAndMetadataTests.java

```diff
@@ -0,0 +1,355 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.ingest;
+
+import org.elasticsearch.index.VersionType;
+import org.elasticsearch.test.ESTestCase;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Set;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasEntry;
+import static org.hamcrest.Matchers.notNullValue;
+
+public class IngestSourceAndMetadataTests extends ESTestCase {
+
+    IngestSourceAndMetadata map;
+
+    @Override
+    public void setUp() throws Exception {
```

Comment:
this seems unnecesary, maybe leftover from debugging?

Reply:
Removed.